crates/rover-client/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "rover-client"
+description = "an http client for making graphql requests for the rover CLI"
+version = "0.0.0"
+authors = ["Apollo Developers <opensource@apollographql.com>"]
+edition = "2018"
+
+[dependencies]
+anyhow = "1.0"
+graphql_client = "0.9"
+http = "0.2.1"
+reqwest = { version = "0.10.7", features = ["json", "blocking"] }
+thiserror = "1.0"
+serde = "1.0.116"
```

```suggestion
serde = "1"
```

serde i go even further and leave out the minor release since i really trust them to not deliver breaking changes. not a hill i'll die on though